.github/workflows/rubyonrails.yml

```diff
@@ -17,7 +17,6 @@ jobs:
         ports:
           - "5432:5432"
         env:
-          POSTGRES_DB: hostedgpt_test
           POSTGRES_USER: rails
           POSTGRES_PASSWORD: password
```

Comment:
In the docker compose, you removed the db, user, and password from the postgres container since it all comes from rails. That feels like a good change. But here within this github CI you only removed the db but kept user and password. Can’t we remove those too since they will also come from the rails container?

Reply:
oh no in the docker compose the user and password are still there. Rails needs a user to connect to the database server in order to create the database so we have to at least set password. https://hub.docker.com/_/postgres the "Environment Variables" section explains these for the postgres image

Reply:
What about now that you removed the USER and PASSWORD from the dockerfile, do we remove them from here?

Reply:
Maybe I made changes after you made this comment. I'm not sure where I removed those. Do you mean the docker-compose.yml file?